app/src/main/java/com/example/playlistmaker/history/SearchHistotyAdapter.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.playlistmaker.history
+
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.playlistmaker.Track
+import com.example.playlistmaker.TrackViewHolder
+
+class SearchHistotyAdapter : RecyclerView.Adapter<TrackViewHolder>() {
```

Comment:
🍏 Опечатка в названии
```suggestion
class SearchHistoryAdapter : RecyclerView.Adapter<TrackViewHolder>() {
```